addons/mail/static/tests/chatter_tests.js

```diff
@@ -624,6 +624,81 @@ QUnit.test('chatter: post, receive and star messages', function (assert) {
         });
 });
 
+QUnit.test('chatter: post a message disable the send button', function(assert) {
+    assert.expect(3); // nombre total de test
+
+    var messages = [];
+    var bus = new Bus();
+    var getSuggestionsDef = $.Deferred();
+    var form = createView({
+        View: FormView,
+        model: 'partner',
+        data: this.data,
+        arch: '<form string="Partners">' +
+                '<sheet>' +
+                    '<field name="foo"/>' +
+                '</sheet>' +
+                '<div class="oe_chatter">' +
+                    '<field name="message_ids" widget="mail_thread" options="{\'display_log_button\': True}"/>' +
+                '</div>' +
+            '</form>',
+        res_id: 2,
+        mockRPC: function (route, args) {
+            if (args.method === 'message_get_suggested_recipients') {
+                return $.when({2: []});
+            }
+            if (args.method === 'get_mention_suggestions') {
+                getSuggestionsDef.resolve();
+                return $.when([{email: "test@odoo.com", id: 1, name: "Test User"}]);
+            }
+            return this._super(route, args);
+        },
+        session: {},
+        intercepts: {
+            get_messages: function (event) {
```

Comment:
Use `ev` instead of `event`.
This is because Google Chrome uses global `event` object, so the code may work in Chrome but not in other browsers.

Reply:
I am using firefox and I did not had any problem during my test. Also I would rather keep the same parameter name as in the other tests.

Reply:
You had no issue in firefox, because you wrote the test correctly.
In those cases, the events are OdooEvent, so it won't rely on the global `event` object.
However, if it was using a browser built-in event, such as MouseEvent, you would have an issue with the following code:

```
get_messages: function () {
   event.stopPropagation();
   ...
}
```

It would work on Google Chrome because it automatically defines the global `event` object, but it may crash on other browsers that do not implement this global `event` object.